services/navigation/builtin/builtin.go

```diff
@@ -181,7 +183,16 @@ func (conf *Config) Validate(path string) ([]string, error) {
 	for _, obs := range conf.Obstacles {
 		for _, geoms := range obs.Geometries {
 			if !geoms.TranslationOffset.ApproxEqual(r3.Vector{}) {
-				return nil, errors.New("geometries specified through the navigation are not allowed to have a translation")
+				return nil, errGeomWithTranslation
+			}
+		}
+	}
+
+	// Ensure bounding regions have no translation
+	for _, region := range conf.BoundingRegions {
+		for _, geoms := range region.Geometries {
+			if !geoms.TranslationOffset.ApproxEqual(r3.Vector{}) {
+				return nil, errGeomWithTranslation
```

Comment:
Is there a test for this new validation?

Reply:
I will add this as a unit test.